app/src/main/java/org/oppia/app/viewmodel/ViewModelBridgeFactory.kt

```diff
@@ -3,16 +3,15 @@ package org.oppia.app.viewmodel
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import javax.inject.Inject
-import javax.inject.Provider
 
 /**
  * Provides a Dagger bridge to facilitate [ViewModel]s supporting @Inject constructors. Adapted from:
  * https://proandroiddev.com/dagger-2-on-android-the-simple-way-f706a2c597e9 and
  * https://github.com/tfcporciuncula/dagger-simple-way.
  */
 class ViewModelBridgeFactory<V : ViewModel> @Inject constructor(
-  private val viewModelProvider: Provider<V>
-): ViewModelProvider.Factory {
+  private val viewModelProvider: javax.inject.Provider<V>
+) : ViewModelProvider.Factory {
   override fun <T : ViewModel?> create(modelClass: Class<T>): T {
     val viewModel = viewModelProvider.get()
     // Check whether the user accidentally switched the types during provider retrieval. ViewModelProvider is meant to
```

Comment:
Update this file with develop, so as to remove this file from this PR for comparison.

Reply:
resolved